src/main/java/com/meta/cp4m/ServiceConfiguration.java

```diff
@@ -19,26 +19,33 @@ public class ServiceConfiguration {
   private final String handler;
   private final @Nullable String store;
   private final String plugin;
+  private final @Nullable String[] preProcessors;
 
   @JsonCreator
   ServiceConfiguration(
       @JsonProperty("webhook_path") String webhookPath,
       @JsonProperty("handler") String handler,
       @JsonProperty("store") @Nullable String store,
-      @JsonProperty("plugin") String plugin) {
+      @JsonProperty("plugin") String plugin,
+      @JsonProperty("pre_processors") @Nullable String[] preProcessors) {
     Preconditions.checkArgument(
         webhookPath != null && webhookPath.startsWith("/"),
         "webhook_path must be present and it must start with a forward slash (/)");
     this.webhookPath = webhookPath;
     this.handler = Objects.requireNonNull(handler, "handler must be present");
     this.store = store;
     this.plugin = Objects.requireNonNull(plugin, "plugin must be present");
+    this.preProcessors = preProcessors;
```

Comment:
```suggestion
    this.preProcessors = preProcessors == null ? Collections.emptyList() : preProcessors;
```

You'll need to also make preProcessors a list before this will work